ddtrace/span.py

```diff
@@ -433,17 +433,20 @@ def set_exc_info(self, exc_type, exc_val, exc_tb):
         if self._ignored_exceptions and any([issubclass(exc_type, e) for e in self._ignored_exceptions]):  # type: ignore[arg-type]  # noqa
             return
 
-        self.error = 1
-
         # get the traceback
         buff = StringIO()
         traceback.print_exception(exc_type, exc_val, exc_tb, file=buff, limit=20)
         tb = buff.getvalue()
+        exc_msg = stringify(exc_val)
+        self.set_exc_fields(exc_type, exc_msg, tb)
 
+    def set_exc_fields(self, exc_type, exc_msg, tb):
```

Comment:
Should propose this change in a separate PR if it is necessary.

Reply:
Yup I plan to cherry pick this commit if we want to add this method. I just wanted to avoid opening up more graphql PRs until we aligned on the implementation details.